packages/protocol/contracts/common/TransferWhitelist.sol

```diff
@@ -79,7 +84,7 @@ contract TransferWhitelist is Ownable, UsingRegistry {
       i = i.add(1);
     }
     for (uint256 j = 0; j < registeredContracts.length; j = j.add(1)) {
-      _whitelist[i] = registry.getAddressFor(registeredContracts[j]);
```

Comment:
@asaj Can you remind me what the original reason was we didn't want to use getAddressForOrDie?